### DISCUSSION TITLE
Ralink wifi dongle doesn't work, any help would be appreciated.

### DISCUSSION THREAD

**Discussion by Abdalnablse10** [devmfc/debian-on-amlogic#104] [Q&A]
Hi I hope everyone is having a lovely day
I wanted to connect a usb 148f:5370 RT5370 wifi dongle but it didn't work, I thought it was a simple missing firmware but the rt2800usb kernel module is missing.

Any help with this would be appreciated.

**Comment by Adney** [comment]
Download kernelheadrs and make module. Easy.

**Reply by Abdalnablse10** [reply]
I know the basics, I just don't know how to correctly "make" modules taken from part of the kernel source tree using the existing Makefile, I wasn't able to find a standalone repo, I might be stupid so please bear with me.
